week 05/netflix-OOP-student/classes/User.class.php

```diff
@@ -0,0 +1,102 @@
+<?php
+	require_once("Security.class.php");
+
+	/**
+	 * 
+	 */
+	class User {
+		private $email;
+		private $password;
+		private $passwordConf;
+		
+	    /**
+	     * @return mixed
+	     */
+	    public function getPasswordConf() {
+	        return $this->passwordConf;
+	    }
+
+	    /**
+	     * @param mixed $passwordConf
+	     *
+	     * @return self
+	     */
+	    public function setPasswordConf($passwordConf) {
+	        $this->passwordConf = $passwordConf;
+
+	        return $this;
+	    }
+
+	    /**
+	     * @return mixed
+	     */
+	    public function getEmail() {
+	        return $this->email;
+	    }
+
+	    /**
+	     * @param mixed $email
+	     *
+	     * @return self
+	     */
+	    public function setEmail($email) {
+	        $this->email = $email;
+
+	        return $this;
+	    }
+
+	    /**
+	     * @return mixed
+	     */
+	    public function getPassword()
+	    {
+	        return $this->password;
+	    }
+
+	    /**
+	     * @param mixed $password
+	     *
+	     * @return self
+	     */
+	    public function setPassword($password)
+	    {
+	        $this->password = $password;
+
+	        return $this;
+	    }
+
+	    public function isAvailable($email) {
+	    	try{
+				$conn = new PDO("mysql:host=localhost;dbname=netflix", "root", "");

```

Comment:
Hier heb je twee keer een databankconnectie. Beter om van in het begin met een DB-klasse te werken ipv telkens de connectie te definiëren.